### PR TITLE
KAN-28: fixed bug with Event card displaying

### PR DESCRIPTION
**Pull request by lestraza**

### CODE REVIEW

File: src/features/scheduler/components/SchedulerMonthRows.tsx [hunk 1]
```diff
@@ -91,32 +90,36 @@ export const SchedulerMonthRows = React.memo(
                         (event) => event.type === EventType.Holiday
                       )?.color || ""
                     }
-                    onClick={() =>
-                      onOpenCard?.({
+                    onClick={() => {
+                      return onOpenCard?.({
```
Comment (stahlhammer88) [inline src/features/scheduler/components/SchedulerMonthRows.tsx]: why return?

File: src/features/scheduler/components/SchedulerMonthRows.tsx [hunk 2]
```diff
@@ -91,32 +90,36 @@ export const SchedulerMonthRows = React.memo(
...
-                          day: item,
-                          eventType: event.type,
-                          userEvent: event,
-                        })
-                      }
-                      key={item.date + event.name}
-                    />
-                  ))}
+                  {item.userEvents.map((event) => {
+                    return (
```
Comment (stahlhammer88) [inline src/features/scheduler/components/SchedulerMonthRows.tsx]: here too

Comment (lestraza) [reply src/features/scheduler/components/SchedulerMonthRows.tsx]: ok

File: src/features/scheduler/components/SchedulerMonthRows.tsx (hunk 1 above)
Comment (lestraza) [reply src/features/scheduler/components/SchedulerMonthRows.tsx]: I have set console.log there)), I'll remove that

**Review by stahlhammer88 — APPROVED** [review]

**Pull request merged**